Tests/RxCocoaTests/UITabBarController+RxTests.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  UITabBarController+RxTests.swift
+//  Tests
+//
+//  Created by Yusuke Kita on 2016/12/07.
+//  Copyright © 2016 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+
+import Foundation
+
+import RxSwift
+import RxCocoa
+import UIKit
+import XCTest
+
+class UITabBarControllerTests: RxTest {
+    let createSubject: () -> UITabBarController = { UITabBarController() }
+}
+
+/**
+ iOS only
+ */
+#if os(iOS)
+extension UITabBarControllerTests {
+    func testWillBeginCustomizing() {
+        let subject = createSubject()
+        
+        let viewControllers = [UIViewController()]
+        var returnedViewControllers: [UIViewController]!
+        
+        _ = subject.rx.willBeginCustomizing
+            .subscribe(onNext: { vc in
+                returnedViewControllers = vc
+            })
```

Comment:
It would be great to add before `subject.delegate!.tabBarController!(subject, willBeginCustomizing: viewControllers)` `XCTAssertNotEqual(returnedViewControllers, viewControllers)`.

I'll need to add those checks everywhere where they aren't present. We actually had a bug because those checks weren't there :)